validators/types_unix.go

```diff
@@ -28,7 +28,9 @@ import (
 var DefaultSysSpec = SysSpec{
 	OS: "Linux",
 	KernelSpec: KernelSpec{
-		Versions: []string{`^3\.[1-9][0-9].*$`, `^([4-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}, // Requires 3.10+, or newer
+		// 4.4 & 4.19 are selected as kernel Super Long Term Support (SLTS), and the Civil Infrastructure Platform will provide support until at least 2026.
+		Versions:     []string{`^4\.4.*$`, `^4\.19.*$`, `^4\.[2-9][0-9].*$`, `^([5-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`},
+		VersionsNote: "Recommended LTS versions from the 4.x series are 4.4 and 4.19. Any 4.x version newer than 4.19 is also supported. For cgroups v2 support, the minimal version is 4.15 and the recommended version is 5.8+",
```

Comment:
for this note, do we have a check on cgroupv2 that the minimal version is satisfied?

cc: @harche

Reply:
The current statement is from 
1. https://github.com/opencontainers/runc/blob/ad5b481dace5cda8ca7c659b7717a15517333198/docs/cgroup-v2.md?plain=1#L23-L24 
2.  https://kubernetes.io/docs/concepts/architecture/cgroups/#requirements: `Linux Kernel version is 5.8 or later`

If the runc one is not accurate, we may change to `the minimal version is 5.8`

Reply:
I'm fine with the message. Just was wondering if we need to add a validaiton in kubelet on start when cgroupv2 are used.

Reply:
there are many kernel version validations in kube. Mainly related to some FG or sysctl.

We may add a warning first for kubelet using cgoup v2.

Reply:
https://github.com/kubernetes/kubernetes/pull/126595